docs/Building-for-Windows.md

```diff
@@ -0,0 +1,18 @@
+# How to build browser for Windows
+Many of the build scripts are bash scripts and not natively invokable in Windows. To solve this, you can build the project using Git Bash.
+1. Install [Git & Git Bash for Windows](https://git-scm.com/downloads)
```

Comment:
On Windows 10, you also need to enable "Developer Mode" so that symbolic links work.
https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/
https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development